src/devices/Ssd1351/ssd1351Imaging.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
```

Comment:
NIT: looks like this file name needs the first letter capitalized.

Reply:
@joperezr 
Renamed in the latest PR update